daffodil-lib/src/main/scala/org/apache/daffodil/util/Memoize1.scala

```diff
@@ -52,6 +52,7 @@ class Memoize1[-T <: AnyRef, +R] private (f: T => R) extends (T => R) {
       case None => {
         val y = f(x)
         vals.put(x, y)
+        Memoize1.count += 1
```

Comment:
can revert all changes to Memoize1. Came in handy while experimenting, but code doesn't need to use it.